boards/arc/qemu_arc/qemu_arc_em.yaml

```diff
@@ -11,3 +11,4 @@ testing:
   ignore_tags:
     - net
     - bluetooth
+vendor: snps
```

Comment:
Let's be consistent: either use snps for all ARC qemu targets, or none (use 'qemu' instead).

Reply:
@ruuddw 

FYI: I've changed ARC QEMU DTS here https://github.com/zephyrproject-rtos/zephyr/pull/62919 to have it consistent.